amazon-kinesis-client-multilang/src/main/java/software/amazon/kinesis/multilang/config/MultiLangDaemonConfiguration.java

```diff
@@ -300,7 +301,7 @@ private AwsCredentialsProvider resolveCredentials(BuilderDynaBean credsBuilder)
     }
 
     private void updateCredentials(BuilderDynaBean toUpdate, AwsCredentialsProvider primary,
-            AwsCredentialsProvider secondary) {
+                                   AwsCredentialsProvider secondary) {
```

Comment:
Please revert this edit. Horizontal alignment provides zero objective benefits, and invites both merge conflicts and worthless git history rewrites.